### PR TITLE
Adds a popup when the help button is clicked.

### PR DESCRIPTION
**Pull request by Astrejoe**
#### What have I done
issue [7] Help #19 
Before, the help button did nothing. I adapted it such that it opens a pop-up when the help button is clicked. It also has a close button for closing the pop-up. I did this using angular material, a module for angular. This required me to add the module as well as a style-sheet to the project.

#### Testing
Click the help button in the top right corner of UI (little question mark.) It should open a pop-up. When you click the close button in the pop-up, it should close again.

### CODE REVIEW

File: src/app/toolbar/help/modal/modal.component.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { Component, OnInit } from '@angular/core';
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+
+@Component({
+    selector: './app-help-modal',
+    templateUrl: './modal.component.html',
+    styleUrls: ['./modal.component.scss']
+})
+export class ModalComponent implements OnInit {
```
Comment (Extremelyd1) [inline src/app/toolbar/help/modal/modal.component.ts]: This line gives the following error: `ModalComponent is declared in multiple Angular modules: TestModule and AppModule`.

File: src/app/toolbar/help/help.component.spec.ts [hunk 2]
```diff
@@ -1,7 +1,19 @@
...
-import { NO_ERRORS_SCHEMA } from '@angular/core';
+import { NgModule, NO_ERRORS_SCHEMA } from '@angular/core';
+import { ModalComponent } from './modal/modal.component';
+import { NgbModule } from '@ng-bootstrap/ng-bootstrap';
+
+// This module is created in order to add ModalComponent to entryComponents
+// when configuring the testBed. The reason for this is that
+// TestBed.configureTestingModule doesn't support entryComponents.
+@NgModule({
+    declarations: [ ModalComponent ],
```
Comment (Extremelyd1) [inline src/app/toolbar/help/help.component.spec.ts]: See comment regarding error.

File: src/app/app.module.ts [hunk 3]
```diff
@@ -22,6 +23,7 @@ import { ImportComponent } from './toolbar/import/import.component';
         ViewportComponent,
         ToolbarComponent,
         HelpComponent,
+        ModalComponent,
```
Comment (Extremelyd1) [inline src/app/app.module.ts]: See comment regarding error.

File: package-lock.json [hunk 4]
```diff
@@ -129,6 +129,23 @@
                 "tslib": "^1.9.0"
```
Comment (Dacaspex) [inline package-lock.json]: `package-lock` has not been updated after you removed the `angular/material` library I believe. Perhaps run `npm install` to update the `package-lock.json`

Comment (Astrejoe) [reply package-lock.json]: I ran npm install. It did remove the material package from package-lock.json, but it also changed and added some stuff to package-lock.json.

**Pull request merged**